src/strategies/BeefyAeroStrategy.sol

```diff
@@ -0,0 +1,313 @@
+    }
+
+    /**
+     * @notice return asset equivalent token0 and token1 ratio in removing liquidity
+     */
+    function _getAssetsAmountFromLP() internal view returns (uint256) {
+        uint256 lpTokenAmount = _getTotalLpTokenAmount();
+        (uint256 token0Out, uint256 token1Out) = aeroRouter.quoteRemoveLiquidity(
+            address(token0), address(token1), false, aeroRouter.defaultFactory(), lpTokenAmount
+        );
+
+        return token0Out + _getSwapPrice(token1, token0, token1Out);
+    }
+
+    /**
+     * @notice swap token from to token to
+     * @param from token to swap
+     * @param to token swapped into
+     * @param amount amount of from token
+     * @param slippage max acceptable slippage in swap
+     */
+    function _swapToken(ERC20 from, ERC20 to, uint256 amount, uint256 slippage) internal {
+        uint256 tokenToAmount = _getSwapPrice(from, to, amount);
+        uint256 minTokenToReceive = _calculateSlippageAmount(tokenToAmount, slippage, true);
+
+        IAeroRouter.Route[] memory route = new IAeroRouter.Route[](1);
+
+        route[0].from = address(from);
+        route[0].to = address(to);
+        route[0].stable = false;
+        route[0].factory = aeroRouter.defaultFactory();
+
+        aeroRouter.swapExactTokensForTokens(amount, minTokenToReceive, route, address(this), block.timestamp);
+    }
+
+    function _provideLiquidityToAero(uint256 token0Amount, uint256 token1Amount, uint256 slippage) internal {
+        (uint256 token0Desired, uint256 token1desired,) = aeroRouter.quoteAddLiquidity(
+            address(token0), address(token1), false, aeroRouter.defaultFactory(), token0Amount, token1Amount
+        );
+
+        uint256 minToken0Amount = _calculateSlippageAmount(token0Desired, slippage, true);
+        uint256 minToken1Amount = _calculateSlippageAmount(token1desired, slippage, true);
+        aeroRouter.addLiquidity(
+            address(token0),
+            address(token1),
+            false,
+            token0Desired,
+            token1desired,
+            minToken0Amount,
+            minToken1Amount,
+            address(this),
+            block.timestamp
+        );
+    }
+
+    function _removeLiquidityFromAero(uint256 lpTokenAmount, uint256 slippage) internal {
+        (uint256 token0Out, uint256 token1Out) = aeroRouter.quoteRemoveLiquidity(
+            address(token0), address(token1), false, aeroRouter.defaultFactory(), lpTokenAmount
+        );
+
+        uint256 minToken0Out = _calculateSlippageAmount(token0Out, slippage, true);
+        uint256 minToken1Out = _calculateSlippageAmount(token1Out, slippage, true);
+
+        aeroRouter.removeLiquidity(
+            address(token0),
+            address(token1),
+            false,
+            lpTokenAmount,
+            minToken0Out,
+            minToken1Out,
+            address(this),
+            block.timestamp
+        );
+    }
+
+    function _investIntoBeefy(uint256 assets, uint256 slippage) internal {
```

Comment:
The pools are balanced in terms of dollar value (like a normal uniswap v2 pool). We don't need to take any ratios. We can just trade half of the assets away.

Reply:
dropped, used 50/50 ratio.